gradle/wrapper/gradle-wrapper.properties

```diff
@@ -5,6 +5,6 @@
 
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-all.zip
```

Comment:
when you realise you've been on default `-bin` distro all along, missing out on IDE docs/code-completion 😩

time to re-config [renovate](https://github.com/leotm/react-native-template-typescript/pull/325/files)